giskard/ml_worker/exceptions/giskard_exception.py

```diff
@@ -1,3 +1,40 @@
 class GiskardException(Exception):
     def __init__(self, *args: object) -> None:
         super().__init__(*args)
+
+
+class GiskardPythonEnvException(GiskardException):
+    def __init__(self, *args: object) -> None:
+        super().__init__(*args)
+
+
+class GiskardPythonVerException(GiskardPythonEnvException):
+    def __init__(self, cls_name, e, *args: object) -> None:
+        super().__init__(
+            f"Failed to load '{cls_name}' due to {e.__class__.__name__}.\n"
+            "Make sure you are loading it in the environment with matched Python version.",
```

Comment:
Can we add the Python version from the pickle meta, it will help user to fix issue

Reply:
Yes, that's what we are considering as well